osu.Game.Rulesets.Mania/Edit/DrawableManiaEditorRuleset.cs

```diff
@@ -16,6 +16,8 @@ namespace osu.Game.Rulesets.Mania.Edit
 {
     public class DrawableManiaEditorRuleset : DrawableManiaRuleset
     {
+        protected override bool DisplayComboCounter => false;
```

Comment:
Is this just for the editor? If so can't it be done in `ManiaEditorPlayfield`?

Reply:
I moved it to `DrawableManiaRuleset` since `ManiaPlayfield` on its own shouldn't display a combo counter by default (see `TestSceneBarLine` for example). I agree it's awkward though, I intended it to be a temporary property that would be removed once something like `LegacyRulesetSpecificCombo` exists.

Reply:
I still don't quite understand. The only time this is set to `false` is in `DrawableManiaEditorRuleset`, which also overrides `CreatePlayfield` and creates the playfield without the combo counter. Am I missing something?